expansion.bzl

```diff
@@ -0,0 +1,561 @@
+"""
+Module containing functions that aid in environment variable expansion.
```

Comment:
hm, this all feels like should be part of a bazel library somewhere.

Reply:
Interesting idea. Not sure where you could have gotten that notion. 😇